base/int.jl

```diff
@@ -57,18 +57,28 @@ typemin(typeof(x))`, `abs(x) == x`, not `-x` as might be expected.
 abs(x::Signed) = flipsign(x,x)
 
 """
+    Base.checked_abs(x)
+
+The absolute value of `x`, with overflow error trapping where applicable.
+For types for which no overflow can happen during `abs`, this is equivalent
+to `abs(x)`.
+"""
+function checked_abs end
```

Comment:
Above is how you attach it to the function, you definitely should not create a dummy method to attach to.

IIUC docstrings are no longer attached to methods (they were previously), but just appended to the function docstring. There is a lot of confusion about this at the moment so I think it ought to be cleared up (hopefully with your issue :) )